layout-server/src/main/java/org/finos/vuu/layoutserver/CorsConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.finos.vuu.layoutserver;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class CorsConfig implements WebMvcConfigurer {
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**")
+                .allowedOrigins("http://127.0.0.1:5173")
```

Comment:
this works for showcase, but not the sample app.
can we add "https://localhost:8443", that will work for the default Vuu deployment

Should probably to think about making this a configuration option.

Reply:
Done: https://github.com/finos/vuu/pull/970/commits/b55f1e42e08daa4c99f34ef55e8bc7ab0f577de5